src/styles/main.css

```
@@ -0,0 +1,273 @@
+  }
+}
+
+.container {
+  box-sizing: content-box;
+  max-width: 1016px;
+  padding: 0 22px;
+  margin: 0 auto;
+  height: 100%;
+}
+
+@media (min-width: 535px) {
+  .container {
+    padding: 0 72px;
+  }
+}
+
+@media (min-width: 535px) and (min-width: 745px) {
+  .container {
+    padding: 0 124px;
+  }
+}
+
+.icon {
+  position: relative;
+  background-size: cover;
+  background-position: center;
+  background-repeat: no-repeat;
+}
+
+.icon--phone {
+  display: block;
+  width: 18px;
+  height: 18px;
+  background-image: url(../images/phone.svg);
+  text-decoration: none;
+}
+
+.icon--phone:hover .museum-phone {
+  opacity: 1;
+}
+
+@media (min-width: 535px) {
+  .icon--phone {
+    width: 24px;
+    height: 24px;
+  }
+}
+
+.icon--menu {
+  display: block;
+  width: 18px;
+  height: 12px;
+  background-image: url(../images/menu.svg);
+}
+
+@media (min-width: 535px) {
+  .icon--menu {
+    width: 24px;
+    height: 16px;
+  }
+}
+
+.top-actions {
+  display: grid;
+  grid-template-columns: 64px 1fr 48px;
+  justify-content: space-between;
+  align-items: center;
+  justify-items: end;
+  height: max-content;
+}
+
+@media (min-width: 535px) {
+  .top-actions {
+    grid-template-columns: 64px 1fr 87px;
+  }
+}
+
+.top-actions--menu {
+  grid-template-columns: 64px 1fr;
+}
+
+.logo {
+  width: 64px;
+  color: #fff;
+  font-family: "Cinzel", sans-serif;
+  font-size: 32px;
+  font-weight: 700;
+  text-decoration: none;
+  line-height: 26px;
+}
+
+@media (min-width: 535px) {
+  .logo {
+    font-size: 40px;
+    line-height: 32px;
+  }
+}
+
+.menu {
+  background-color: #d12d35;
+  height: 100vh;
+}
+
+@media (min-width: 745px) {
+  .menu {
+    height: 800px;
+  }
+}
+
+.menu__top {
+  margin-bottom: 40px;
+}
+
+.menu__content {
+  box-sizing: border-box;
+  padding: 25px 0;
+}
+
+.menu__list {
+  margin: 0 0 32px;
+  padding: 0;
+  list-style: none;
+}
+
+.menu__link {
+  color: #fff;
+  font-size: 22px;
+  line-height: 22px;
+  font-weight: 700;
+  letter-spacing: 3.5px;
+  text-decoration: none;
+  text-transform: uppercase;
+}
+
+.menu__item {
+  margin-bottom: 32px;
+}
+
+.menu__item:last-child {
+  margin-bottom: 0;
+}
+
+.menu__call {
+  width: max-content;
+  margin-top: 16px;
+  padding-bottom: 8px;
+  border-bottom: solid white 1px;
+  color: #fff;
+  font-size: 12px;
+  font-weight: 700;
+  line-height: 16px;
+  letter-spacing: 1.5px;
+  text-transform: uppercase;
+}
+
+.locations-phone {
+  width: 201px;
+  height: 108px;
+  color: #fff;
+  text-align: right;
+  font-weight: bold;
+  line-height: 27px;
+  text-transform: uppercase;
+  opacity: 0;
+  pointer-events: none;
+}
+
+.locations-phone__5ht-avenue {
+  margin-bottom: 8px;
+}
+
+.locations-phone--header {
+  position: absolute;
+  top: -47px;
+  right: 36px;
+}
+
+.locations-phone--menu {
+  opacity: 1;
+  text-align: left;
+}
+
+.page__menu {
+  position: fixed;
+  top: 0;
+  left: 0;
+  right: 0;
+  transform: translateX(-100%);
+  transition: transform 0.3s;
```

create variable for this
```suggestion
  transition: transform $transitionDuration;
```